### ISSUE TITLE
No sound of call 14.4.2

### ISSUE THREAD

**Issue by DmitriiPl** [LNSSPsd/PodsGrant#39]
hi. there is no sound when calling 12pro max 14.4.2 untether with app2 5B58. I am ready to give you any files you say. thanks

**Comment by DmitriiPl** [comment]
and the volume control also works

**Comment by DmitriiPl** [comment]
[Bluetooth.zip](https://github.com/LNSSPsd/PodsGrant/files/12187260/Bluetooth.zip)

**Comment by plia7** [comment]
> [Bluetooth.zip](https://github.com/LNSSPsd/PodsGrant/files/12187260/Bluetooth.zip)

Hi what is the file link? Will it work with Fogu 14 unc0over unteather jailbreak for iPhone 12 Pro Max iOS 14.4.1?

**Comment by DmitriiPl** [comment]
No, these are log files. I posted them to be watched by the developer of Tweak




пятница, 28 июля 2023 г., 02:41 +0300 от ***@***.***  ***@***.***>:

**Comment by thexYeti** [comment]
> Im tested without tweak microphone and sound of call works on 13.2.2 14.71 14.8. But not work on 14.4.2. But strange that ipad with 14.7.1 - work sound and iphone 12promax with 14.7.1 - not work sound. Tweak not help. Is it possible for the tweak to fix the problem with sound on 14.4.2?


Could you reiterate, does your call audio and mic work on 14.8? 

My mic doesn't work at all on 14.8.1 :(

**Comment by plia7** [comment]
> > Im tested without tweak microphone and sound of call works on 13.2.2 14.71 14.8. But not work on 14.4.2. But strange that ipad with 14.7.1 - work sound and iphone 12promax with 14.7.1 - not work sound. Tweak not help. Is it possible for the tweak to fix the problem with sound on 14.4.2?
> 
> Could you reiterate, does your call audio and mic work on 14.8?
> 
> My mic doesn't work at all on 14.8.1 :(

I got air pods gen 3, and installed PodsGrant latest version 0.3.0
for my iPhone 12 Pro Max iOS 14.4.1
spatial audio is enabled and working I can see it under settings.
the volume control there is a tweak Silquo Pro for that
noise cancellation seem to work too when I tested a call with it the quality is better
so if your airpods gen 1 or 2 don't work well on iOS 14.8.1 I suggest you get airpods gen 3
